cmd/podman/machine/machine_nonwindows.go

```diff
@@ -0,0 +1,12 @@
+//go:build !windows
```

Comment:
Can you call this file machine_unix.go


podman (VENDOR) $ find . -name \*nonwindows.go | wc -l
1
podman (VENDOR) $ find . -name \*unix.go | wc -l
102

Reply:
Yes I was overthinking this a bit. _unix would be a smaller subset (linux || aix || darwin || dragonfly || freebsd || netbsd || openbsd || solaris || zos || hurd || ios || illumos) than what machine.go is using (amd64 || arm64). However in practice this distinction doesn't matter since I don't think we care about plan 9.

Reply:
Yes we will deal with other OS when they come about.